src/v3/src/types/schema.ts

```diff
@@ -515,6 +515,22 @@ export interface ImageWithTextElement extends UISchemaElement {
   };
 }
 
+export interface ImageLinkElement extends UISchemaElement {
+  type: 'ImageLink';
+  options: {
+    id: string;
+    url: string;
+    altText: string;
+    alignment: 'start' | 'center' | 'end';
+    svgIcon: FunctionComponent<{
+      description: string;
+      width?: string | number;
+      height?: string | number;
+    }>,
+    dataSe: string;
```

Comment:
Just a few more style nits with this interface (don't think `id` is needed and then some props can probably be optional) and then I think we're good to go:
```suggestion
    href: string;
    altText: string;
    alignment?: 'start' | 'center' | 'end';
    SVGIcon: FunctionComponent<{
      description: string;
      width?: string | number;
      height?: string | number;
    }>,
    dataSe?: string;
```

Reply:
please see https://github.com/okta/okta-signin-widget/pull/3526/commits/7d077eeb9f3ad0341503ec173531a5b6c65c907e